src/WebJobs.Extensions.DurableTask/IDurabilityProviderFactory.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    /// <summary>
+    /// Interface defining methods to build orchestration services and orchestration service clients.
```

Comment:
Should this comment reference the DurabilityProvider type instead of orchestration services and clients?